### ISSUE TITLE
VideoClips.get_clip to provide to users some clipId and a way to get meta information about clip pts/time in video

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/vision#3340]
It would be nice for debugging to have VideoClips provide not only `video_idx` but also `clip_idx`. With that index one should have a way to get the temporal location corresponding to the sampled clip.

This allows to correlate the sampled clip with subtitles or just scroll into the video for debugging. Currently the returned `info` object just has `video_fps` property, `clip_idx` is not there.

**Comment by bjuncek** [comment]
Yeah, I agree - I think there was an issue regarding this a while back but we decided that a finer grained API would be more useful than this - given that fine-grained API will be in beta for a while (at least one more release) it might be useful to add this

Do you want to send a PR with the changes?
If not, I can take a look into this late next week

**Comment by vadimkantorov** [comment]
> Do you want to send a PR with the changes?

busy with the research project :( don't have resources for truthful involvement :(

**Comment by bjuncek** [comment]
@vadimkantorov yeah I get it. I hacked my way around it to use it with HowTo100M as well. 
If it's not crucial, I'll get some time allocated to this after ICCV then :)
